common/cmap_test.go

```diff
@@ -0,0 +1,67 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	cmap = NewCMap()
+)
+
+func TestIterateKeysWithValues(t *testing.T) {
+	cmap.Clear()
```

Comment:
Self-contained tests are better in my view. Let's move `cmap = NewCMap()` inside each test